Improve command documentation (#314)
kyma-project/cli: docs/gen-docs/kyma_test_run.md

### Options

```
      --concurrency int    Number of tests to be executed in parallel. (default 1)

>   -c, --count int          Defines how many times every test should be executed. "count" and "max-retries" flags are mutually exclusive. (default 1)

Reviewer: ```suggestion
  -c, --count int          Number of times every test should be executed. "count" and "max-retries" flags are mutually exclusive. (default 1)
```

>       --max-retries int    Defines how many times a given test is retried when it fails. A suite is marked with a "succeeded" status even if some tests failed at first and then finally succeeded. The default value of 0 means that there are no retries of a given test.

Reviewer: ```suggestion
      --max-retries int    Number of times a given test is retried when it fails. A suite is marked with a "succeeded" status even if some tests failed at first and then finally succeeded. The default value of 0 means that there are no retries of a given test.
```
